samples/vue-query/src/api/endpoints/petstoreFromFileSpecWithTransformer.ts

```diff
@@ -49,7 +49,7 @@ export const listPets = (
 export const getListPetsQueryKey = (
   params?: MaybeRef<ListPetsParams>,
   version = 1,
-) => [`/v`, version, `/pets`, ...(params ? [params] : [])] as const;
+) => ['v', version, 'pets', ...(params ? [params] : [])] as const;
```

Comment:
@Maxim-Mazurok 

> RE: I did it this way because I wasn't sure how Vue Query hashes the key, I thought that if it does [].join("") for example - then /v1/air/plane would have the same key as /v1/airplane which would be a mistake.

I've changed what functionality you had here previously. Is this what you're describing as a requirement? (forward slash removed). Any string concatenation with variables even in the pathname should be cut up and recognised as a var. 
eg: `/something${var}/pets/${petId}` would explode to `['something',var,'pets',petId]`

Reply:
It's all good now, thank you. I was just confirming that we can get rid of slashes and keep arrays just like you did, all good currently 👍